pos_partner_deselection/doc/index.rst

```diff
@@ -1,25 +1,29 @@
-==================
- Deselect Partner
-==================
+===================
+Partner deselection
+===================
+
+Installation
+============
+
+* `Install <https://odoo-development.readthedocs.io/en/latest/odoo/usage/install-module.html>`__ this module in a usual way
 
 Configuration
 =============
 
 * Go to ``Point of Sale >> Configuration >> Point of Sale``
 
-  * Select a POS
-  * Click on ``[Edit]``
+  * Select a Point of Sale (POS)
+  * Click on ``[Edit]`` and then select  ``[Settings]``
+  * Go to the Features section
   * Specify a **Customer Deselection Interval** in seconds
   * Click ``[Save]``
 
 Usage
 =====
 
-{Instruction for daily usage. It should describe how to check that module works. What shall user do and what would user get.} 
-
-* Open a POS
+* Open a Point of Sale (POS)
 * Click ``Customers``
 * Select a customer
 * Click ``Set Customer``
-* Wait time you set in configuration
-* RESULT: Customer is Deselected
+* Wait for an amount time which you set up in Deselection Interval field.
+* **RESULT:** Customer is Deselected after some amount of time
```

Comment:
RESULT in bold looks cool, but we need to update template then to use it everywhere: https://odoo-development.readthedocs.io/en/latest/dev/docs/usage-instructions.html

Reply:
follow the Template for now :)